python/xorbits/_mars/dataframe/groupby/len.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+import pandas as pd
+
+from ... import opcodes
+from ...core import OutputType
+from ...core.operand import Operand, OperandStage
+from ..operands import DataFrameOperandMixin
+
+
+class GroupByLen(DataFrameOperandMixin, Operand):
+    _op_type_ = opcodes.GROUPBY_LEN
+
+    def __call__(self, groupby):
+        return self.new_scalar([groupby])
+
+    @classmethod
+    def tile(cls, op: "GroupByLen"):
+        in_groupby = op.inputs[0]
+
+        # generate map chunks
+        map_chunks = []
+        for chunk in in_groupby.chunks:
+            map_op = op.copy().reset_key()
+            map_op.stage = OperandStage.map
+            map_op.output_types = [OutputType.series]
+            chunk_inputs = [chunk]
+
+            map_chunks.append(map_op.new_chunk(chunk_inputs))
+
+        # generate reduce chunks, we only need one reducer here.
+        out_chunks = []
+        reduce_op = op.copy().reset_key()
+        reduce_op.output_types = [OutputType.scalar]
+        reduce_op.stage = OperandStage.reduce
+        out_chunks.append(reduce_op.new_chunk(map_chunks))
```

Comment:
```suggestion
        out_chunks.append(reduce_op.new_chunk(map_chunks, shape=(), index=(0,), dtype=op.outputs[0].dtype))
```

Reply:
really helped and solved the problem, much appreciated!